test/e2e/sched/sched_test.go

```diff
@@ -308,3 +309,16 @@ func findKubeSchedulerProfileByName(sc *schedconfig.KubeSchedulerConfiguration,
 
 	return nil, nil
 }
+
+func schedulerConfigurationFromData(data []byte) (*schedconfig.KubeSchedulerConfiguration, error) {
```

Comment:
let's use `manifests.DecodeSchedulerProfilesFromData`, and remove the dep on the scheduler config apis (which transitively pull `k8s.io/kubernetes`)